app/scripts/services/hpa.js

```diff
@@ -99,6 +100,27 @@ angular.module("openshiftConsole")
     };
 
 
+    var hasV2HPAAnnotations = function(hpaResources) {
+      return _.some(hpaResources, function(hpa) {
+        return $filter('hpaMetrics')(hpa);
```

Comment:
Since this is temporary and we shouldn't need it after we update to the newer API, I'd just inline the full annotation here.

```js
return $filter('annotation')(hpa, 'autoscaling.alpha.kubernetes.io/metrics');
```

I'd rather not do more than we need to with the annotations if the same values are supported in the v2beta1 API. And then you don't need the common PR.

nit: better to resolve the filter outside the loop

Reply:
This makes sense as well, closed the other PR.